onnx/test/automatic_downgrade_test.py

```diff
@@ -6,7 +6,8 @@
 import numpy as np
 from automatic_conversion_test_base import TestAutomaticConversion
```

Comment:
Here as well. Import automatic_conversion_test_base and use automatic_conversion_test_base.TestAutomaticConversion